experiments/rollouts_from_policies_config.csv

```diff
@@ -0,0 +1,13 @@
+config_name,env_name,n_demonstrations
```

Comment:
Curious how you chose these numbers, but they seem reasonable (10 for simple tasks, 25 for most MuJoco, 240 for Humanoid).

Reply:
With the exception of the custom envs, I used the maximum number of demonstrations used in the GAIL paper's benchmarks (almost every env had three datapoints with 3 different # of demos).

For the ant custom envs I just copied the original ant env's # of demos.

For the 2DMaze I thought 10 would be fine, but did 25 just in case. (The imitation learning phase can choose the number of demonstrations to use from disk)